Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN pip install --upgrade pip wheel jmespath-terminal
 # openssh - included for ssh-keygen
 # ca-certificates 
 # wget - required for installing jp
-RUN apk update && apk add bash gcc openssl-dev libffi-dev musl-dev jq openssh ca-certificates wget openssl && update-ca-certificates
+RUN apk update && apk add bash gcc openssl-dev libffi-dev musl-dev jq openssh ca-certificates wget openssl && update-ca-certificates git
```

Comment:
This looks iffy - shouldn't you `apk add git`? As it is now, you'd run `update-ca-certificates git` which I don't know what it'd do :)

same for changes in Dockerfile...